node/forest_libp2p/src/service.rs

```diff
@@ -128,8 +128,8 @@ where
             warn!("Failed to bootstrap with Kademlia: {}", e);
         }
 
-        let (network_sender_in, network_receiver_in) = channel(20);
-        let (network_sender_out, network_receiver_out) = channel(20);
+        let (network_sender_in, network_receiver_in) = channel(200);
```

Comment:
Too high, no? Or did you try lower and the channels were choking out?

Reply:
oh ooops, didn't mean to swap